### ISSUE TITLE
Use HealthcareService resource to pull only Locations that allow direct scheduling

### ISSUE THREAD

**Issue by jbalboni** [department-of-veterans-affairs/va.gov-team#9072]
Instead of the various eligibility checks and VATS flags for direct scheduling, in VSP we will fetch all the available HealthcareServices for a given Organization, and filter the locations out from that.

This will make it so you can't create requests when va_online_scheduling_vsp_appointment_new is turned on. Evaluating how to have requests and VSP DS flow on at the same time will be covered in a separate ticket.

AC (only applies when va_online_scheduling_vsp_appointment_new is on)
- [ ] Function created in Location resource that fetches health care services by type of care and organization, and includes location 
- [ ] Locations returned are used to generate the facilities show to user on the facility page
- [ ] Mock data from v1 api added to `services/healthcareService` folder
- [ ] `services/healthcareService` functions accept feature flag and use it to choose which api to call
- [ ] Action creators are updated to pass feature flag in where appropriate
- [ ] Action creators and reducers are updated with any changes needed to accommodate service differences

**Comment by jbalboni** [comment]
Blocked by https://issues.mobilehealth.va.gov/browse/VAOSR-613
